configs/modules.config

```diff
@@ -230,4 +230,29 @@ process {
     //         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     //     ]
     // }
+    // MITOHIFI
+    // withName: 'FINDMITOREF' {
+    //     ext.args   = '--species${meta.species} --email${meta.email} --min_length${meta.min_length}'
+    // TO DO:    not sure what to enter as prefix as this should be the path to the directory that is mounted
+    //     ext.prefix = { ? }
+    //     publishDir = [
+    //         path: { "$params.outdir/mitohifi/findmitoref" },
+    //         mode: params.publish_mode,
+    //         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+    //     ]
+    // }
+    // withName: 'MITOHIFI' {
+    //     ext.args   = '-t 1 -o 5'
+    // TO DO:    not sure what to enter as prefix as it has different values at the moment: 
```

Comment:
This needs to be fixed by fixing the input channels and the input files for MitoHiFi (see assemblies from prepare_input and output from FINDMITOREF)